plenum/test/replica/test_buffers_cleaning.py

```diff
@@ -0,0 +1,63 @@
+from plenum.server.replica import Replica
+
+
+class FakeNode():
```

Comment:
I can do it if you insist on, but I see no real reason to start a whole node instead of using just a plain object here.